hash_practice/exercises.py

```diff
@@ -2,18 +2,84 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: On^4 (?)
+        Space Complexity: On
     """

```

Comment:
This works, but do you need all these nested loops?  Can you see a way to do it using a dictionary, maybe with the key being the letters common in the words.